source/pages/Settings/PrivateKey.tsx

```diff
@@ -91,14 +95,35 @@ const PrivateKeyView = () => {
         <CopyCard
           className="my-3"
           onClick={
-            valid ? () => copyText(String(activeAccount?.xprv)) : undefined
+            valid
+              ? () =>
+                  copyText(
+                    String(
+                      isSyscoinChain
+                        ? CryptoJS.AES.decrypt(
+                            activeAccount?.xprv,
+                            pwd
+                          ).toString()
+                        : activeAccount?.xprv
+                    )
+                  )
+              : undefined
           }
           label="Your private key"
         >
           <p>
-            {valid
-              ? ellipsis(activeAccount?.xprv, 4, 16)
-              : '********...************'}
+            {
+              // eslint-disable-next-line
+              valid && isSyscoinChain
+                ? ellipsis(
+                    CryptoJS.AES.decrypt(activeAccount?.xprv, pwd).toString(),
```

Comment:
we can create a method to decrypt instead of making verifications and calling the same method twice in jsx, to be easier to read